scripts/docker-entrypoint.sh

```diff
@@ -297,7 +297,7 @@ if [ -e "$context_xml" ]; then
   cp "$context_xml" "${CATALINA_HOME}"/conf/
 else
     if [ "$SECRET_DB_USERNAME" == "" ] ; then
-      echo "DB_USERNAME must be specified. Assuming this as a passwordless connection.";
+      echo "As DB_USERNAME is not specified, trying password less connection.";
```

Comment:
since this is client-facing>>
No DB_USERNAME specified; trying connection without a password, which is not recommended.

Reply:
`which is not recommended` Should we add it? Because there is no recommendation from us to use password-less or password based!!

Reply:
@taz-pega-work password-less connection doesn't mean unauthenticated here, authentication could be managed by Kerberos or some other external method.